src/hooks/useAutomateConnection.ts

```diff
@@ -28,8 +28,12 @@ export async function isConnectedToAutomate(ethereum: any): Promise<boolean> {
         },
       ],
     });
-    return res.client === 'automate';
+    if (res.client !== 'automate') {
+      return 'notConnectedToAutomate';
```

Comment:
Why are you returning `notConnectedToAutomate` here and `none` below? There's no difference between those cases. Just throw here and it will be caught below